[npm ERR! notarget] Version 9.6.2 is not at npm

### Reporting a bug?

Wanting to update my packages but getting the following Error:
```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @intlify/core-base@9.6.2.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

### Expected behavior

npm update should work fine

### Reproduction

`npm update`

### System Info

```shell
System:
    OS: Linux 5.15 Ubuntu 22.04.1 LTS 22.04.1 LTS (Jammy Jellyfish)
    CPU: (4) arm64 Neoverse-N1
    Memory: 21.23 GB / 23.43 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
Binaries:
    Node: 18.18.2 - ~/.nvm/versions/node/v18.18.2/bin/node
    npm: 9.8.1 - ~/.nvm/versions/node/v18.18.2/bin/npm
```
```


### Screenshot

_No response_

### Additional context

_No response_

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/intlify/vue-i18n-next/blob/master/.github/CONTRIBUTING.md)
- [X] Read the [Documentation](https://vue-i18n.intlify.dev/)
- [X] Check that there isn't [already an issue](https://github.com/intlify/vue-i18n-next/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussions](https://github.com/intlify/vue-i18n-next/discussions)

today i also have same issue on Cloud Worker pineline  : 

21:55:31.954 | Success: Finished cloning repository files
21:55:32.777 | Detected the following tools from environment: npm@9.6.7, nodejs@18.17.1
21:55:32.779 | Installing project dependencies: npm install --progress=false
21:55:49.260 | npm ERR! code ETARGET
21:55:49.261 | npm ERR! notarget No matching version found for @intlify/core-base@9.6.2.
21:55:49.261 | npm ERR! notarget In most cases you or one of your dependencies are requesting
21:55:49.261 | npm ERR! notarget a package version that doesn't exist.
21:55:49.264

any idea ?

--- a/netlify.toml
+++ b/netlify.toml
@@ -1,6 +1,0 @@
-[build.environment]
-  NODE_VERSION = "16"
-  NPM_FLAGS = "--version"
-[build]
-  publish = "docs/.vitepress/dist"
-  command = "npx pnpm@8 i --store=node_modules/.pnpm-store && npx pnpm@8 run docs:build"

--- a/package.json
+++ b/package.json
@@ -167,7 +167,7 @@
       "secretlint"
     ]
   },
-  "packageManager": "pnpm@8.10.0",
+  "packageManager": "pnpm@8.9.2",
   "pnpm": {
     "overrides": {
       "vue": "3.3.4",

--- a/renovate.json
+++ b/renovate.json
@@ -13,7 +13,8 @@
     "@intlify/vue-i18n-core",
     "vue-i18n",
     "petite-vue-i18n",
-    "vue-i18n-bridge"
+    "vue-i18n-bridge",
+    "pnpm"
   ],
   "major": {
     "automerge": false

